tutorials/embedded-c-getting-started/index.md

```diff
@@ -0,0 +1,297 @@
+You’ll need to have the following for ESP-IDF to work properly:
+
+ - Python 3.5 or higher
+ - Git
+ - Cmake
+ - Ninja
+
+If you don't have these dependencies, you can install them using :
+
+**For Mac :**
+```
+brew install python
+brew install git
+pip install ninja
+pip install cmake
+```
+
+**For Windows :**
+
+Python : https://www.python.org/downloads/windows/
+Git : https://git-scm.com/download/win
+ninja : https://github.com/ninja-build/ninja/releases
+cmake : https://cmake.org/download/
+
+Once you have all dependencies installed, configure ESP-IDF.
+
+1. Select your git and python version
+1. Select the location you want to download ESP-IDF
+1. Click the download button to download the ESP-IDF tools
+1. Run the tool check to verify your installation
+
+If the tool check verification succeeds, you’re ready to continue.
+
+Once ESP-IDF is completely installed, try out the hello-world example to see if everything is working properly, I suggest putting the command to initialize ESP-IDF into an alias:
+
+`alias get_idf='. $HOME/esp/esp-idf/export.sh'`
+
+in your $HOME/.profile file so you can just call get_idf. If you don't have a profile dotfile, then put the code above in $HOME/.bash_profile.
+For more troubleshooting steps, see the [getting started](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/) page of the ESP-IDF.
+
+### ESP32 Setup
+
+We will be using the Espressif Systems ESP32 (ESP32), which is an inexpensive and easy to source microcontroller with WiFi and Bluetooth capabilities. To run this example, you will need an LED and two wires to connect it to the board if the LED is tolerant of the same voltage as the board(typically 3.3v or 5v) if it can’t then you should use a resistor in series with the [LEDS](http://www.resistorguide.com/resistor-for-led/).
+
+The ESP32 will communicate with Google Cloud IoT Core using Wifi and will send telemetry data using the MQTT protocol, we will also read the internal temperature sensor to send telemetry data to the device's subscription topic.
+
+To get the internal temperature we will use the temprature_sens_read function. To correctly set the function you must give a forward declaration for the function:
+```
+#ifdef __cplusplus
+
+extern "C" {
+  #endif
+
+uint8_t temprature_sens_read();
+
+#ifdef __cplusplus
+}
+#endif
+```
+
+### Google Cloud IoT Core
+
+If you’ve never used IoT Core, don’t worry, the steps below will get you setup to transmit telemetry data to the cloud but before we can do that lets talk about IoT Core and its components. Google Cloud IoT Core is a complete set of tools to connect, process, store, and analyze data both at the edge and in the cloud. Google Cloud IoT consists of the device management API for creating and managing logical collections of devices and the protocol bridge which adapts device-friendly protocols (MQTT or HTTP) to scalable Google infrastructure.
```

Comment:
Standardize everything to use "IoT Core" rather than "Google Cloud IoT Core" and "Google Cloud IoT".